schema.sql

```diff
@@ -84,4 +84,7 @@ CREATE TABLE visits (
     PRIMARY KEY (id),
     FOREIGN KEY (vet_id) REFERENCES vets(id) ON DELETE CASCADE,
     FOREIGN KEY (animal_id) REFERENCES animals(id) ON DELETE CASCADE
-);
\ No newline at end of file
+);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Comment:
- [ ] Kindly add the indexes in the `schema.sql` file, not only show them in the screenshots.